tests/python/relay/opencl_texture/test_conv2d_nhwc_texture.py

```diff
@@ -398,11 +384,9 @@ def test_conv2d_inceptionv3_nhwc_3c():
 
 
 @tvm.testing.requires_opencl
-def test_conv2d_1x1_16c16spatial():
-    target = "opencl --device=adreno"
-    dtype = "float16"
-
-    input_shape = (1, 256, 256, 16)
+@tvm.testing.parametrize_targets("opencl -device=adreno")
+def test_conv2d_1x1_16c16spatial(target, dtype):
+    input_shape = (1, 128, 128, 16)
```

Comment:
Why did you change input shape?

Reply:
Because there was an error that there is not enough memory for opencl